vumi/transports/truteq/tests/test_truteq.py

```diff
@@ -280,8 +280,9 @@ def test_handle_inbound_sms(self):
             yield self.send(cmd)
             [warning] = logger.messages()
             self.assertEqual(
-                warning,
-                "Received unsupported message, dropping: %r." % (cmd,))
+                warning[:59],
+                "Received unsupported message, dropping: <MO command_id=103 ")
+        self.flushLoggedErrors()
```

Comment:
The `repr()` of the command object contains the value of a dict. CPython and pypy serialise the fields in different orders, so we can't assert on the whole string.